lldb/include/lldb/API/SBSaveCoreOptions.h

```diff
@@ -53,6 +54,27 @@ class LLDB_API SBSaveCoreOptions {
   /// \return The output file spec.
   SBFileSpec GetOutputFile() const;
 
+  /// Set the process to save, or unset if supplied with a null process.
```

Comment:
change `null process` to a `default constructed SBProcess`... though Maybe we should require a valid process here and add a `void ClearProcess()` API to have this make more sense.